tests/unit/handlers/test_container_engine.py

```diff
@@ -53,4 +53,4 @@ def test_format(self):
         }
         payload = handler.format(record)
 
-        self.assertEqual(payload, json.dumps(expected_payload))
+        self.assertEqual(payload, json.dumps(expected_payload, ensure_ascii=False))
```

Comment:
I think this would be better as a separate test, rather than overriding the old `test_format`. You should duplicate this test and call it `test_format_unicode` or something

Also, it would be better to do an `self.assertEqual(payload['message'], message)`. That's a more direct test than relying on `json.dumps` to do what we expect

Reply:
payload after `handler.format(record)` will be a `String`

How about like this?

```
expected_payload = json.dumps({
            "message": message,
            "timestamp": {"seconds": 5, "nanos": int(0.03 * 1e9)},
            "thread": record.thread,
            "severity": record.levelname,
        }, ensure_ascii=False)

        payload = handler.format(record)

        self.assertEqual(payload, expected_payload)
```

Thank you

Reply:
Hmm good point. I'd probably prefer to check for a substring then: `self.assertTrue(message in payload)`

But if that doesn't work for some reason, I think your original code is fine. Thanks for the contribution!

Reply:
Hi @Daniel-Sanche 

Thank you!

I learned a lot from here :)